### PR TITLE
feat(ui): displaying user logs

### PR DESCRIPTION
**Pull request by mitchelvanbever**
Attempt to display console logs added by the user.

* Also adjusted the panels structure to ensure percentage based heights allow for more responsiveness

See #346 for more information

### CODE REVIEW

**Review by DerYeger — APPROVED** [review]

File: packages/ui/client/components/ConsoleOutput.vue [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <div
+      flex justify-between cursor-pointer bg-header h-auto all:transition-400 p="x4 y2"
+      @click="$emit('update:modelValue', !modelValue)"
+    >
+      <p> Console output ({{ consoleOutput?.length || 0 }}) </p>
+      <IconButton icon="i-carbon-caret-down" :class="!modelValue ? 'rotate-180' : 'rotate-0'" :disabled="!logs?.length" />
+    </div>
+    <template v-if="modelValue">
+      <div p="x4 y6">
+        <div v-for="log of consoleOutput" :key="log" my-2>
```
Comment (Demivan) [inline packages/ui/client/components/ConsoleOutput.vue:31]: Idea: Add indication for stdout vs stderr logs

Comment (mitchelvanbever) [reply packages/ui/client/components/ConsoleOutput.vue:31]: Good suggestion, I'm wrapping everything in pre now to have some formatting will add the use of `text-orange` if the `log.type` is `stderr`

File: packages/ui/client/components/FileDetails.vue [hunk 2]
```diff
@@ -55,15 +56,11 @@ const changeViewMode = (view: Params['view']) => {
         </button>
       </div>
     </div>
-    <ViewModuleGraph v-show="viewMode === 'graph'" :graph="graph" class="file-details-graph" />
-    <ViewEditor v-if="viewMode === 'editor'" :file="current" />
-    <ViewReport v-else-if="!viewMode" :file="current" />
+    <div flex flex-col flex-1 overflow="hidden">
+      <ViewModuleGraph v-if="viewMode === 'graph'" :graph="graph" class="file-details-graph" />
```
Comment (DerYeger) [inline packages/ui/client/components/FileDetails.vue:60]: I forgot to remove the now unused class `file-details-graph` 😅

**Comment by mitchelvanbever** [comment]
I think it's ready for review 😁 

I'm open to suggestions on how to display it better, but keep in mind all that we're getting is a string.
Also I'm unsure if this is the best way to go about of storing the logs, feel free to change the implementation (or tell me what to change 🙂).

How it looks currently:
Google Chrome-Vitest_2022-01-11

**Pull request merged**